backend/app/infrastructure/external_api/bitquery/blocks_reward.py

```diff
@@ -0,0 +1,43 @@
+import requests
+
+import app as app_module
+from app.constants import BITQUERY_API
+from app.exceptions import ExternalApiException
+from app.extensions import w3
+from app.infrastructure.external_api.bitquery.req_producer import (
+    produce_payload,
+    BitQueryActions,
+    get_bitquery_header,
+)
+
+
+def accumulate_blocks_reward_wei(blocks: list) -> int:
+    blocks_reward_gwei = 0.0
+    for block in blocks:
+        blocks_reward_gwei += float(block["reward"])
+
+    return int(w3.to_wei(blocks_reward_gwei, "gwei"))
+
+
+def get_blocks_reward(address: str, start_time: str, end_time: str) -> int:
```

Comment:
Don't we have the same issue with dates as we did with block numbers (the same inclusive value for end in epoch `n` and start in epoch `n + 1`)?